src/overlays/actors/ovl_Bg_Jya_Lift/z_bg_jya_lift.c

```diff
@@ -26,24 +36,128 @@ const ActorInit Bg_Jya_Lift_InitVars = {
     (ActorFunc)BgJyaLift_Destroy,
     (ActorFunc)BgJyaLift_Update,
     (ActorFunc)BgJyaLift_Draw,
+
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Jya_Lift/func_80899BA0.s")
+static InitChainEntry sInitChain[] = {
+    ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_CONTINUE),
+    ICHAIN_F32(unk_F4, 1400, ICHAIN_CONTINUE),
+    ICHAIN_F32(unk_F8, 1800, ICHAIN_CONTINUE),
+    ICHAIN_F32(unk_FC, 2500, ICHAIN_STOP),
+};
+extern s32      D_0600D7E8;
+extern Gfx      D_0600CCE0[];
+
+
+    void BgJyaLiftInitDynapoly(BgJyaLift* this, GlobalContext* globalCtx, u32 arg2, DynaPolyMoveFlag moveFlag) {
```

Comment:
looks like this is indented when it shouldnt be. The format script should talke care of stuff like this. make sure to run ./format.sh
youll need to install clang-tidy and clang-format before doing so